src/DynamoCore/Graph/Workspaces/Serialiaztion/Workspaces.cs

```diff
@@ -1,54 +1,18 @@
-﻿using CoreNodeModels;
-using CoreNodeModels.Input;
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
 using Dynamo.Core;
 using Dynamo.Engine;
-using Dynamo.Graph.Nodes;
-using Dynamo.Graph.Nodes.CustomNodes;
 using Dynamo.Graph.Nodes.NodeLoaders;
 using Dynamo.Graph.Workspaces;
+using Dynamo.Models;
 using Dynamo.Scheduler;
 using Newtonsoft.Json;
-using PythonNodeModels;
-using System;
-using System.Collections.Generic;
-using System.Text.RegularExpressions;
 
 namespace Autodesk.Workspaces
 {
     public static class Utilities
     {
-        private static Dictionary<string, Type> NodeNameMap = new Dictionary<string, Type>()
```

Comment:
okay, so the approach you took is to remove the compile time requirement of workspace serialization on the core node models, and instead make this a runtime requirement?

Does this actually run 😉 ?